build/Targets/Settings.props

```diff
@@ -141,8 +146,6 @@
 
   <!-- Windows specific settings -->
   <PropertyGroup Condition="'$(OS)' == 'Windows_NT'">
-    <BaseNuGetRuntimeIdentifier>win7</BaseNuGetRuntimeIdentifier>
```

Comment:
Pretty sure this was a workaround for a bug that got fixed a long time ago, where things didn't work properly for anycpu configs.

Reply:
Yes. Kill this with fire.

Reply:
You _might_ be able to kill more of this with fire as well...